Framework/Algorithms/src/FitPeaks.cpp

```diff
@@ -922,6 +928,11 @@ void FitPeaks::processInputPeakTolerance() {
   m_minSignalToNoiseRatio = getProperty(PropertyNames::PEAK_MIN_SIGNAL_TO_NOISE_RATIO);
   if (isEmpty(m_minSignalToNoiseRatio) || m_minSignalToNoiseRatio < 0.)
     m_minSignalToNoiseRatio = 0.;
+
+  // set the signal-to-sigma threshold to zero (default value) if not specified or invalid
+  m_minSignalToSigmaRatio = getProperty(PropertyNames::PEAK_MIN_SIGNAL_TO_SIGMA_RATIO);
+  if (isEmpty(m_minSignalToSigmaRatio) || m_minSignalToSigmaRatio < 0.)
```

Comment:
Could it ever be empty given the default value in `declareProperty`?

Reply:
I guess the user could set it to empty using `alg.setProperty(EMPTY_INT())`. I've added this check because it is used consistently with other parameters in the algorithm even with ones that has default values